mongodbatlas/fw_resource_mongodbatlas_search_deployment.go

```diff
@@ -31,12 +31,13 @@ var _ resource.ResourceWithImportState = &SearchDeploymentRS{}
 
 const (
 	searchDeploymentDoesNotExistsError = "ATLAS_FTS_DEPLOYMENT_DOES_NOT_EXIST"
+	searchDeploymentName               = "search_deployment"
```

Comment:
nit: this looks like a separate PR? no need to do it now, but consider it for the next time

Reply:
Yes true. Tried to keep this PR as clean as possible with respect to the resource implementation, but you usually some pending details always get adjusted when implementing the data source.